modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -765,6 +791,42 @@ class AbsToleranceScalar : public WrappableScalar<AbsToleranceScalar>
     double _tol;
 };
 
+class AbsToleranceRect : public WrappableRect<AbsToleranceRect>
```

Comment:
Intersection over union?

Reply:
replaced this version of comparison by IoU. Thanks for advise!